lib/apis/user_api_manager.dart

```diff
@@ -127,4 +128,26 @@ class UserApiManager extends BaseApiManager {
       success(GetAllAddress.fromJson(result.data).addresses);
     }
   }
+
+  static Future<void> getAllCities(Function success, Function fail) async {
```

Comment:
I totally recommend separating addresses APIs and City Area APIs 
each of them in a stand-alone file

Reply:
I separated addresses APIs to AddressApiManager
and  City Area APIs to AddressAssetApiManager